adafruit_wsgi/wsgi_app.py

```diff
@@ -71,7 +71,7 @@ def __call__(self, environ: Dict[str, str], start_response: Callable):
             except (ValueError, TypeError) as err:
                 raise RuntimeError(
                     "Proper HTTP response return not given for request handler '{}'".format(
```

Comment:
Can you change this from "request handler" to "request handler for path" to match the change?  Thanks!

Reply:
Done, made this an f-string too.

Reply:
Did you get an error for that?  Some of the M0 boards don't actually support f-strings as of now, but this module also relies on `re` which I think means this library already doesn't support them... @dhalbert, do you have an opinion?  Either way, the CI shouldn't have raised this an error (@kattni).

Reply:
I would not expect this library to be used with M0 boards, so assume you can use `re` and f-strings.

Reply:
Going to resolve this - it's not an f-string, or format. It's simply a '+' as it's already a string too.